sass/_navigation.scss

```diff
@@ -221,3 +192,36 @@ a {
         line-height: normal;
     }
 }
+
+.menu .menu-item-has-children {
+    position: relative;
+    
+    .sub-menu-toggle {
+        cursor: pointer;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets